content/references-rest-api.md

```diff
@@ -61,6 +61,107 @@ Event data
 | channel | true | A single channel name. *Limited to 100 channels.* |
 | socket_id | false | Exclude a connection from the targets of the event. |
 
+## Channels
+
+### GET channel
+
+Channel writes a JSON object to the response with information about the
```

Comment:
This part should contain details about the purpose of the endpoint and not describe what it does.